src/w3c/templates/headers.js

```diff
@@ -129,14 +129,12 @@ export default (conf, options) => {
                 >${conf.thisVersion}</a
               >
             </dd>
-            <dt>${l10n.latest_published_version}</dt>
-            <dd>
-              ${conf.latestVersion
-                ? html`<a href="${conf.latestVersion}"
-                    >${conf.latestVersion}</a
-                  >`
-                : "none"}
-            </dd>
+            ${conf.latestVersion
+              ? html`<dt>${l10n.latest_published_version}</dt>
+                  <dd>
+                    <a href="${conf.latestVersion}">${conf.latestVersion}</a>
+                  </dd>`
+              : ""}
```

Comment:
Should we keep the old behavior of showing "none" as latest version or skip it altogether?

Reply:
Oh, good point! Yeah, “none” is good.